docs/tracking-algorithms/index.rst

```diff
@@ -16,3 +16,9 @@ For full details of the API for each of the components please see the
    TSTORMS <tstorms>
    Tempest Extremes <tempest_extremes>
    TRACK <track>
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Other
+
+   Preprocessing Data <preprocessing_data>
```

Comment:
I think I'd be inclined to create a new section called `Data` or similar for details on preprocessing and output description.

I feel this is more to do with wider usage than specific algorithms.

Thoughts?